producer_client/spec/lib/animals/resources/cat_spec.rb

```diff
@@ -28,4 +28,38 @@
 
     it_behaves_like 'listing cats', [garfield, felix, sylvester]
   end
+
+  describe '#create' do
+    name = 'Garfield'
+    age = 39
+
+    before do
+      request = {
+        cat: {
+          name: name,
+          age: age,
+        }
+      }
+      response = {
+        data: {
+          id: Pact.like('1'),
+          type: 'cat',
+          attributes: {
+            name: name,
+            age: age,
+          },
+        }
+      }
+      headers = {
+        'Content-Type' => 'application/json',
```

Comment:
I found out the hard way that this header was required. It wasn't immediately obvious, because it's not technically necessary for this test in the client itself. However, for the pact the verify the correct content type must be sent for the producer.